sys-apps/ripgrep/ripgrep-0.10.0.recipe

```diff
@@ -0,0 +1,323 @@
+CHECKSUM_SHA256_70="15f2b5fb00ccdf689e0149d1b1b3c03fead81c2b37735d812fa8bddbbf41b6d8"
+
+SOURCE_URI_71="https://static.crates.io/crates/termcolor/termcolor-1.0.3.crate#noarchive"
+CHECKSUM_SHA256_71="ff3bac0e465b59f194e7037ed404b0326e56ff234d767edc4c5cc9cd49e7a2c7"
+
+SOURCE_URI_72="https://static.crates.io/crates/termion/termion-1.5.1.crate#noarchive"
+CHECKSUM_SHA256_72="689a3bdfaab439fd92bc87df5c4c78417d3cbe537487274e9b0b2dce76e92096"
+
+SOURCE_URI_73="https://static.crates.io/crates/textwrap/textwrap-0.10.0.crate#noarchive"
+CHECKSUM_SHA256_73="307686869c93e71f94da64286f9a9524c0f308a9e1c87a583de8e9c9039ad3f6"
+
+SOURCE_URI_74="https://static.crates.io/crates/thread_local/thread_local-0.3.6.crate#noarchive"
+CHECKSUM_SHA256_74="c6b53e329000edc2b34dbe8545fd20e55a333362d0a321909685a19bd28c3f1b"
+
+SOURCE_URI_75="https://static.crates.io/crates/ucd-util/ucd-util-0.1.1.crate#noarchive"
+CHECKSUM_SHA256_75="fd2be2d6639d0f8fe6cdda291ad456e23629558d466e2789d2c3e9892bda285d"
+
+SOURCE_URI_76="https://static.crates.io/crates/unicode-width/unicode-width-0.1.5.crate#noarchive"
+CHECKSUM_SHA256_76="882386231c45df4700b275c7ff55b6f3698780a650026380e72dabe76fa46526"
+
+SOURCE_URI_77="https://static.crates.io/crates/unicode-xid/unicode-xid-0.1.0.crate#noarchive"
+CHECKSUM_SHA256_77="fc72304796d0818e357ead4e000d19c9c174ab23dc11093ac919054d20a6a7fc"
+
+SOURCE_URI_78="https://static.crates.io/crates/unreachable/unreachable-1.0.0.crate#noarchive"
+CHECKSUM_SHA256_78="382810877fe448991dfc7f0dd6e3ae5d58088fd0ea5e35189655f84e6814fa56"
+
+SOURCE_URI_79="https://static.crates.io/crates/utf8-ranges/utf8-ranges-1.0.1.crate#noarchive"
+CHECKSUM_SHA256_79="fd70f467df6810094968e2fce0ee1bd0e87157aceb026a8c083bcf5e25b9efe4"
+
+SOURCE_URI_80="https://static.crates.io/crates/version_check/version_check-0.1.4.crate#noarchive"
+CHECKSUM_SHA256_80="7716c242968ee87e5542f8021178248f267f295a5c4803beae8b8b7fd9bc6051"
+
+SOURCE_URI_81="https://static.crates.io/crates/void/void-1.0.2.crate#noarchive"
+CHECKSUM_SHA256_81="6a02e4885ed3bc0f2de90ea6dd45ebcbb66dacffe03547fadbb0eeae2770887d"
+
+SOURCE_URI_82="https://static.crates.io/crates/walkdir/walkdir-2.2.5.crate#noarchive"
+CHECKSUM_SHA256_82="af464bc7be7b785c7ac72e266a6b67c4c9070155606f51655a650a6686204e35"
+
+SOURCE_URI_83="https://static.crates.io/crates/winapi/winapi-0.3.5.crate#noarchive"
+CHECKSUM_SHA256_83="773ef9dcc5f24b7d850d0ff101e542ff24c3b090a9768e03ff889fdef41f00fd"
+
+SOURCE_URI_84="https://static.crates.io/crates/winapi-i686-pc-windows-gnu/winapi-i686-pc-windows-gnu-0.4.0.crate#noarchive"
+CHECKSUM_SHA256_84="ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6"
+
+SOURCE_URI_85="https://static.crates.io/crates/winapi-util/winapi-util-0.1.1.crate#noarchive"
+CHECKSUM_SHA256_85="afc5508759c5bf4285e61feb862b6083c8480aec864fa17a81fdec6f69b461ab"
+
+SOURCE_URI_86="https://static.crates.io/crates/winapi-x86_64-pc-windows-gnu/winapi-x86_64-pc-windows-gnu-0.4.0.crate#noarchive"
```

Comment:
Are these actually used/needed?

Reply:
No; thanks for pointing it out! Seems that you can't list situational dependencies in `Cargo.lock`, hence them being automatically pulled in, but I've included a patch to remove them from it now.

Reply:
Are you sure it builds like this? I have seen these dependencies being pulled in and compiled with cargo as well. Are you sure they won't be pulled in through other means if you don't download them in this phase?